### PR TITLE
feat: add overview price component

### LINKED ISSUE #10481: Create overview attribute price component which shows price and quantity.
Create new component and use in general and bundle overview attribute.

cases of price and quantity display
- Show price only
- Show price, quantity and total price
- Show quantity only

https://4prsat.axshare.com/#id=uvh77c&p=w_cps_summary_1&g=1

**proper formatting of price will be done in https://github.com/SAP/spartacus/issues/10625 for this ticket, just show the value as it is. no formatting, no currency sign**

### PR DESCRIPTION
**Pull request by kacperknapik**
Closes #10481

### CODE REVIEW

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    fixture = TestBed.createComponent(ConfiguratorOverviewPriceComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('overview price type', () => {
+    it('should be PRICE_ONLY', () => {
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.spec.ts]: rather `it('should be PRICE_ONLY' if component carries product price')`,
To make clear what the test really expects under certain conditions. Similar for other test descriptions

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      ><span>{{ calculateTotal() }}</span></ng-container
+    >
+
+    <ng-container *ngSwitchCase="overviewPriceType.QUANTITY_ONLY"
+      ><span>{{ quantity }}</span></ng-container
+    >
+
+    <ng-container *ngSwitchCase="overviewPriceType.PRICE_AND_QUANTITY"
+      ><span
+        >{{ quantity }} {{ productPrice }} {{ calculateTotal() }}</span
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html]: This is not final, right?
We should not display those numbers without any separators or explanations?

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="overview-price">
+  <ng-container [ngSwitch]="getOverviewPriceType()">
+    <ng-container *ngSwitchCase="overviewPriceType.PRICE_ONLY"
+      ><span>{{ calculateTotal() }}</span></ng-container
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html]: Currency support will be added, right?
(Not part of this ticket I guess)

**Review by ChristophHi — CHANGES_REQUESTED** [review]

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html (hunk 2 above)
Comment (kacperknapik) [reply feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html]: Not a part of this ticket. It clearly states, that here we only should add the values, without any styling whatsoever.

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html (hunk 3 above)
Comment (kacperknapik) [reply feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html]: Not a part of this ticket. It clearly states, that here we only should add the values, without any styling whatsoever.

File: feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.spec.ts (hunk 1 above)
Comment (kacperknapik) [reply feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.spec.ts]: Done

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**